examples/playlist/playlist_test.go

```diff
@@ -0,0 +1,155 @@
+package playlist
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/grafana/kindsys"
+	"github.com/grafana/kindsys/pkg/santhoshsys"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRawVersion(t *testing.T) {
+	sys, err := GetRawKind()
+	require.NoError(t, err)
+
+	checkValidVersion(t, sys)
+	checkInvalidVersion(t, sys)
+	checkMigrations(t, sys)
+
+	manifest, err := santhoshsys.CreateResourceKindManifest(sys)
+	require.NoError(t, err)
+	require.NotEmpty(t, manifest)
+	//fmt.Printf("KIND: %s\n", string(manifest))
+}
+
+func TestThemaVersion(t *testing.T) {
+	sys, err := GetThemaKind()
+	require.NoError(t, err)
+
+	checkValidVersion(t, sys)
+
+	// Thema is not yet using the resource version to validate a payload
```

Comment:
What should be SSoT? Kinds or ResourceKinds? 🤔

Reply:
I don't quite understand the question.   The issue pointed to in this test is that a resource says its version:
```
{
  "apiVersion": "playlists.ext.grafana.com/v0-0",
  "kind": "Playlist",
  ...
```
and we need to use that to validate the payload.  

ResourceKinds will represent kinds that can be represented as k8s resources, while "Kinds" is the more generic term that also represents ComposableKinds